src/Entities/Misc/TimedTriggerSpikes.cs

```diff
@@ -108,27 +107,49 @@ private struct SpikeInfo {
 
         private List<MTexture> spikeTextures;
 
+        
+
+        private bool grouped = false;
+        protected bool Grouped {
+            get {
+                return grouped && CommunalHelperModule.maxHelpingHandLoaded;
+            }
+        }
+        protected bool Triggered = false;
+
         public static Entity LoadUp(Level level, LevelData levelData, Vector2 offset, EntityData entityData) {
+            if (entityData.Bool("Grouped", false) && !CommunalHelperModule.maxHelpingHandLoaded) {
+                throw new Exception("Grouped Timed Trigger Spikes attempted to load without Max's Helping Hand as a dependency.");
```

Comment:
Looking at it, probably move this into the EntityData constructor if that still gets caught by the LoadingThread try/catch